index.html

```diff
@@ -253,35 +262,50 @@ <h2 class="style">Multi-Post Stories</h2>
       <h1 class=" hide style">Contact Us</h1>
       <h1 class="lets-chat">Let's chat!</h1>
     </div>
-    <p class="style">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <p class="style">If you have an application you are interested in developing, a feature that you need built or a
+      project that needs coding. I’d love to help with it </p>
     <form class="form-outlay" method="POST" action="https://formspree.io/f/mvodvrgn">
-      <fieldset class="contact-form-fieldset">
-        <ul class="contact-form-ul">
-          <li><input class="input-text style" type="text" name="user_name" placeholder="Francis Ugorji" required maxlength="30"></li>
-          <li><input class="input-email style" type="email" name="user_email" placeholder="francis@gmail.com" required ></li>
-          <li><textarea class=" enter-text style" required maxlength="500" name="user_message">Enter text here</textarea></li>
+      <div class="contact-form-fieldset">
+        <ul class="contact-form-ul display-flex">
+          <li>
+            <label for="user_name"></label>
+            <input id="user_name" class="input-text style" type="text" name="user_name" placeholder="Francis Ugorji"
+              required maxlength="30">
+          </li>
+          <li>
+            <label for="user_email"></label>
+            <input id="user_email" class="input-email style" type="email" name="user_email"
+              placeholder="francis@gmail.com" required>
+
+          </li>
+          <li>
+            <label for="user_message"></label>
+            <textarea id="user_message" placeholder="Enter text here" class="enter-text style" required maxlength="500"
+              name="user_message">Enter text here
+          </textarea>
+          </li>
           <li><button class="see-project submit-button" type="submit">See Project</button></li>
           <li><button class="send-button submit-button" type="submit">Send</button></li>
-        </ul>        
-      </fieldset>
+        </ul>
+      </div>
     </form>
   </section>
   <footer class="social-media-handles display-flex">
     <div class="footer-image-div">
-      <ul class="display-flex justify-content">
-        <li>
+      <ul class="social-icons display-flex justify-content">
+        <li class="icon-frame">
           <a href="#"><img src="Images/Icon-GitHub.png" alt="A Github logo Icon"></a>
         </li>
-        <li>
+        <li class="icon-frame">
           <a href="#"><img src="Images/Icon-Linkedin.png" alt="A LinkedIn logo Icon"></a>
```

Comment:
OPTIONAL

- [ ] Keep the alternative text a summary of the img tag. For this case, Linkedin logo is just enough. You may want to keep it simple for all the alt text.